gammapy/estimators/points/core.py

```diff
@@ -667,13 +667,30 @@ def plot(self, ax=None, sed_type=None, energy_power=0, time_format="iso", **kwar
         flux = scale_plot_flux(flux=flux.to_unit(flux_unit), energy_power=energy_power)
         if "time" in flux.geom.axes_names:
             flux.geom.axes["time"].time_format = time_format
-        ax = flux.plot(ax=ax, **kwargs)
-        ax.set_ylabel(f"{sed_type} [{ax.yaxis.units.to_string(UNIT_STRING_FORMAT)}]")
-        ax.set_yscale("log")
+            ax = flux.plot(ax=ax, **kwargs)
+        else:
+            ax = flux.plot(ax=ax, **kwargs)
+            ax.set_xlabel(f"Energy [{ax.xaxis.units.to_string(UNIT_STRING_FORMAT)}]")
+            ax.set_xscale("log", nonpositive="clip")
+        self._plot_format_yax(ax=ax, energy_power=energy_power, sed_type=sed_type)
+
         if len(flux.geom.axes) > 1:
             ax.legend()
         return ax
 
+    @staticmethod
+    def _plot_format_yax(ax, energy_power, sed_type):
```

Comment:
In general does the `energy_power>0` makes sense if `sed_type!="dnde"`?

Reply:
I would personally say that it does not. But there is no reason to remove it (which would generate API changes). If there would be a use case it is covered